pkg/client/didexchange/client.go

```diff
@@ -96,15 +96,14 @@ func New(ctx provider) (*Client, error) {
 
 // CreateInvitation create invitation
 func (c *Client) CreateInvitation(label string) (*Invitation, error) {
-	verKey, err := c.wallet.CreateEncryptionKey()
+	_, sigPubKey, err := c.wallet.CreateKeyPairSet()
```

Comment:
@sandrask it creates the signing key here now.

Reply:
great